language/en/webpushnotifications_common_acp.php

```diff
@@ -48,6 +48,6 @@
 	'PWA_ICON_LARGE_EXPLAIN'	=> 'File name of a 512px x 512px PNG image. This file must be uploaded to your board’s <samp>' . \phpbb\webpushnotifications\ext::PWA_ICON_DIR . '</samp> directory.',
 	'PWA_ICON_SIZE_INVALID'		=> '“%s” does not have the correct image dimensions.',
 	'PWA_ICON_MIME_INVALID'		=> '“%s” must be a PNG image file.',
-	'PWA_IMAGE_INVALID'			=> '“%s” does not appear to be a valid image file.',
+	'PWA_IMAGE_INVALID'			=> '“%s” does not appear to be a valid image file. Verify the file name and location are correct.',
```

Comment:
This can be thrown an image file is either invalid or doesn't exist. Probably add the latter point here too (like `“%s” does not appear to be a valid image file or the file does not exist in defined location` etc).